scripts/exrrfs_upp.sh

```diff
@@ -7,6 +7,7 @@ cd ${DATA}/${FHR}
 fhr=${FHR:1:2} # remove leading zeros
 CDATEp=$($NDATE ${fhr} ${CDATE} )
 timestr=$(date -d "${CDATEp:0:8} ${CDATEp:8:2}" +%Y-%m-%d_%H.%M.%S)
+timestr2=$(date -d "${CDATEp:0:8} ${CDATEp:8:2}" +%Y-%m-%d_%H:%M:%S)
```

Comment:
Why we need line 10 again?

@EricJames-NOAA Do we still have problems to use the timestr format in line 9?

Reply:
@guoqing-noaa no, we don't need to have line 10...line 9 should work fine.

Reply:
After delete the line 10, the UPP crashed with error information:

```
 im,jm,lm=         1799        1059          59
  Warning TIME 2024-11-06_08.00.00 NOT FOUND in wrf_io.F90, line         417
  Warning in wrf_io.F90, line         704
    Bad time index for DateStr = 2024-11-06_08.00.00
  Warning Status =           -3  in wrf_io.F90, line        3017
  Warning TIME 2024-11-06_08.00.00 NOT FOUND in wrf_io.F90, line         417
  Warning in wrf_io.F90, line         704
    Bad time index for DateStr = 2024-11-06_08.00.00


```

Reply:
@hu5970 let me try to dig into this.  I had thought we could get rid of line 10 now, but let me check what's going on

Reply:
@EricJames-NOAA . I had thought you solved this issue. Thanks for digging further into this.

Reply:
The UPP hash used here is: 72108ee2dab07c

Reply:
@guoqing-noaa do you have a test case with MPAS output?  I am having trouble getting the workflow to run on Jet when I start from scratch, it seems the config is not being read or something.  Ungrib crashes with this message:

ERROR: edition_num: unable to open GRIBFILE.AAA
+ exrrfs_ungrib.sh[77]: export err=0
+ exrrfs_ungrib.sh[77]: err=0
+ exrrfs_ungrib.sh[77]: err_chk
 completed cleanly
++ exrrfs_ungrib.sh[79]: date -d '20240527 03' +%Y-%m-%d_%H
+ exrrfs_ungrib.sh[79]: outfile=IC_PREFIX_not_defined:2024-05-27_03
+ exrrfs_ungrib.sh[80]: [[ -s IC_PREFIX_not_defined:2024-05-27_03 ]]
+ exrrfs_ungrib.sh[92]: echo 'FATAR ERROR: ungrib failed'
FATAR ERROR: ungrib failed

Reply:
@EricJames-NOAA You can find a UPP run directory and grab a case to test:
/mnt/lfs5/BMC/wrfruc/mhu/rrfs2/v2.0.3/stmp/rrfs/v2.0.3/rrfs.20241106/15/upp

Reply:
@EricJames-NOAA Thanks for testing. I did not run Ming's PR.
I think you may try the current rrfs-mpas-jedi branch. If this works, it should work for Ming's PR as well.

Reply:
@guoqing-noaa @hu5970 OK, we do still need line 10.  Alternatively, we could take an approach similar to what is done for David's realtime runs on Jet where we set up itag like this:

DateStr='${fcst_time_str//./:}'

Which replaces . with : for the DateStr in itag.

Reply:
You can see the realtime script here on Jet:

/home/role.wrfruc/HRRRv5/scripts/upp.sh